.github/workflows/phpunit-tests.yml

```diff
@@ -112,28 +112,35 @@ jobs:
   # - Reports test results to the Distributed Hosting Tests.
   # - todo: Configure Slack notifications for failing tests.
   test-php:
-    name: ${{ matrix.php_versions }} on ${{ matrix.os }}
+    name: ${{ matrix.php }}${{ matrix.memcached && ' (with memcached)' || '' }} on ${{ matrix.os }}
```

Comment:
Source: https://github.community/t/do-expressions-support-ternary-operators-to-change-their-returned-value/18114/3